src/tiles/list/tile-list.ts

```diff
@@ -140,7 +140,7 @@ export class TileList extends LitElement {
         ${this.viewsTemplate} ${this.ratingTemplate} ${this.reviewsTemplate}
       </div>
       ${this.topicsTemplate} ${this.collectionsTemplate}
-      ${this.descriptionTemplate}
+      ${this.descriptionTemplate} ${this.textSnippetsTemplate}
```

Comment:
suggestion: add unit tests to cover this method and call

Reply:
List items had no unit tests at all to begin with, so I've added some to cover this and other basic list stuff.